pypeit/spectrographs/ntt_efosc2.py

```diff
@@ -0,0 +1,268 @@
+        #self.meta['dispangle'] = dict(ext=0, card='HIERARCH ESO INS GRIS1 WLEN', rtol=2.0, required_ftypes=['science', 'standard']) did not find dispangle
+        self.meta['idname'] = dict(ext=0, card='HIERARCH ESO DPR CATG')
+    
+    def compound_meta(self, headarr, meta_key):
+        """
+        Methods to generate metadata requiring interpretation of the header
+        data, instead of simply reading the value of a header card.
+
+        Args:
+            headarr (:obj:`list`):
+                List of `astropy.io.fits.Header`_ objects.
+            meta_key (:obj:`str`):
+                Metadata keyword to construct.
+
+        Returns:
+            object: Metadata value read from the header(s).
+        """
+        if meta_key == 'binning':
+            binspatial = headarr[0]['CDELT1']
+            binspec = headarr[0]['CDELT2']
+            binning = parse.binning2string(int(binspec), int(binspatial))
+            return binning
+        elif meta_key == 'decker':
+            try:  # Science
+                decker = headarr[0]['HIERARCH ESO INS SLIT1 NAME']
+            except KeyError:  # Standard!
+                try:
+                    decker = headarr[0]['HIERARCH ESO SEQ SPEC TARG']
+                except KeyError:
+                    return None
+            return decker
+        else:
+            msgs.error("Not ready for this compound meta")
+
+
+    def get_detector_par(self, hdu, det):
+        """
+        Return metadata for the selected detector.
+
+        Args:
+            hdu (`astropy.io.fits.HDUList`_):
+                The open fits file with the raw image of interest.
+            det (:obj:`int`):
+                1-indexed detector number.
+
+        Returns:
+            :class:`~pypeit.images.detector_container.DetectorContainer`:
+            Object with the detector metadata.
+        """
+        # Binning
+        binning = self.get_meta_value(self.get_headarr(hdu), 'binning')  # Could this be detector dependent??
+
+        # Detector 1 (Thor)  -- http://www.eso.org/sci/php/optdet/instruments/fors2/index.html
+        detector_dict = dict(
+            binning         = binning,
+            det             = 1,
+            dataext         = 0,
+            specaxis        = 1,
+            specflip        = False,
+            spatflip        = False,
+            platescale      = 0.126,  # average between order 11 & 30, see manual
+            darkcurr        = 0.0,
+            saturation      = 2.0e5,  # I think saturation may never be a problem here since there are many DITs
+            nonlinear       = 0.80,
+            mincounts       = -1e10,
+            numamplifiers   = 1,
+            gain            = np.atleast_1d(0.70),
+            ronoise         = np.atleast_1d(2.9), # High gain
+            datasec         = np.atleast_1d('[11:2059,:]'),  # For 1x binning, I think
```

Comment:
If this only holds for 1x1 binning (likely), then we should
note in the docs that this is the only mode supported so far.

Reply:
If there are binned modes, it would be better if parsed rather than hardwiring these numbers.

Reply:
I am taking these values directly from VLT/FORS, I have to check more details on this instrument.